packages/flutter_test/lib/src/event_simulation.dart

```diff
@@ -209,7 +209,7 @@ class KeyEventSimulator {
       case 'windows':
         result['keyCode'] = keyCode;
         result['scanCode'] = scanCode;
-        result['characterCodePoint'] = key.keyLabel?.codeUnitAt(0) ?? 0;
+        result['characterCodePoint'] = key.keyLabel.isEmpty ? 0 : key.keyLabel?.codeUnitAt(0);
```

Comment:
Why do we still need `?` after `key.keyLabel` here?

Reply:
Ok I see you've explained it in the PR description. But how come it doesn't trigger lint error?

Reply:
You're right, I don't need the `?`.  Not sure why there's no linter error.